src/bundle/Resources/public/js/scripts/admin.content.tree.js

```diff
@@ -94,6 +95,6 @@
     contentTreeContainer.addEventListener('transitioncancel', handleContainerTransitionStop, false);
     contentTreeContainer.addEventListener('transitionend', handleContainerTransitionStop, false);
 
-    doc.addEventListener('scroll', handleViewportChange, { capture: false, passive: true });
+    contentColumnContainer.addEventListener('scroll', handleViewportChange, { capture: false, passive: true });
```

Comment:
Is this still needed? Won't ContentTree have the same height regardless of scroll position?